src/examples/ensemble.ts

```diff
@@ -0,0 +1,74 @@
+import { SpheroMini } from '../toys/sphero-mini';
+import { wait } from '../utils';
+import { findSpheroMini } from './lib/scanner';
+import { findToys } from './lib/scanner';
+
+import { Core } from '../../toys/core';
+import { Peripheral } from 'noble';
+import * as noble from 'noble';
+
+import { IToyAdvertisement } from '../../toys/types';
+
+const WAIT_TIME: number = 100;
+
+export const ensemble = async (minis: SpheroMini[]) => {
+  var i=0;
```

Comment:
`let i = 0;`

I thought I had a linter :)

Reply:
You do have a linter...it caught me the second time I tried to commit. Not sure why it didn't complain the first time.

All that said, thanks for inviting me to collaborate on this project. Even so, if it's fine with you, I might still submit changes as pull requests. I'm using this project to fuel my interest in coding against Sphero bots, but also to strengthen my TypeScript skills. I'd rather have someone who knows TypeScript better than me review my changes to ensure that I'm not doing something stupid and give me feedback from which I can learn.